apps/gitness/src/pages-v2/repo-commit-details/commit-diffs-container.tsx

```diff
@@ -0,0 +1,72 @@
+import { useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+
+import * as Diff2Html from 'diff2html'
+import { compact } from 'lodash-es'
+
+import { useDiffStatsQuery, useGetCommitDiffQuery } from '@harnessio/code-service-client'
+import { CommitDiffsView } from '@harnessio/ui/views'
+
+import { useGetRepoRef } from '../../framework/hooks/useGetRepoPath'
+import { useTranslationStore } from '../../i18n/stores/i18n-store'
+import { parseSpecificDiff } from '../../pages/pull-request/diff-utils'
+import { PathParams } from '../../RouteDefinitions'
+import { changedFileId, DIFF2HTML_CONFIG, normalizeGitFilePath } from '../pull-request/pull-request-utils'
+import { useCommitDetailsStore } from './stores/commit-details-store'
+
+export default function RepoCommitDiffsPage() {
+  const repoRef = useGetRepoRef()
+  const { commitSHA } = useParams<PathParams>()
+  const { setDiffs, setDiffStats } = useCommitDetailsStore()
+
+  const { data: currentCommitDiffData } = useGetCommitDiffQuery({
+    repo_ref: repoRef,
+    commit_sha: commitSHA || ''
+  })
+
+  const defaultCommitRange = compact(commitSHA?.split(/~1\.\.\.|\.\.\./g))
+  const diffApiPath = `${defaultCommitRange[0]}~1...${defaultCommitRange[defaultCommitRange.length - 1]}`
```

Comment:
This regex keeps repeating across the codebase. Can we pull it out to a constant?

<img width="733" alt="image" src="https://github.com/user-attachments/assets/e340add6-83b9-4d9f-96b1-5168daac9688" />